test/extended/extended_test.go

```diff
@@ -11,15 +11,28 @@ import (
 	_ "github.com/openshift/origin/test/extended/jobs"
 	_ "github.com/openshift/origin/test/extended/router"
 	_ "github.com/openshift/origin/test/extended/security"
+	e2e "k8s.io/kubernetes/test/e2e"
+	"os"
+	"testing"
 
 	exutil "github.com/openshift/origin/test/extended/util"
 )
 
 // init initialize the extended testing suite.
 func init() {
-	exutil.InitTest()
+	// Kubernetes: Pure end to end tests ~ These flags pass through.
+	if os.Getenv("RUN_KUBE_E2E_SUBSET") == "true" {
+		e2e.RegisterFlags()
```

Comment:
he did say no, but I don't think he envisioned a single binary accomplished this way :)